README.md

```diff
@@ -896,6 +896,16 @@ Note that the Circus Train configuration only supports one set of AWS credential
           table-name: aws-a_table
           table-location: s3://<aws-a-data-bucket>/<pata-to-aws-a_db>/aws-a_table
 
+## Circus Train extensions
+
+### Circus Train DataSqueeze Copier
```

Comment:
This and the next would move to level `####`

Reply:
Done.